src/shared/dom.js

```diff
@@ -22,6 +22,20 @@ export function teardownEach ( iterations, detach, start ) {
 	}
 }
 
+export function spreadAttributes ( node, obj ) {
+	if (obj !== null && typeof obj === 'object') {
+		for ( var property in obj ) {
```

Comment:
I think it's reasonable to expect `Object.keys` to exist, especially since it's easily polyfilled if you absolutely have to support IE8 or whatever — though a `for-in` is quicker, no?

(Related: the compiler generates code with `Object.assign` which is less widely-supported than `Object.keys` — again, can be easily polyfilled but I wondered if there should be an internal `assign` helper instead.)